surface/include/pcl/surface/3rdparty/poisson4/octree_poisson.hpp

```diff
@@ -95,8 +97,7 @@ namespace pcl
       }
       if(!children){
         fprintf(stderr,"Failed to initialize children in OctNode::initChildren\n");
-        exit(0);
-        return 0;
+        POISSON_THROW_EXCEPTION (pcl::poisson::PoissonBadInitException, "Failed to initialize OctNode children.");
```

Comment:
I've seen you remove stderr logs in other places but you aren't doing it here. Is there a reason for the inconsistency?

Reply:
My bad, did not intend to leave these.. does github support 🤦‍♂️  reaction yet?